vulcanai/models/basenetwork.py

```diff
@@ -617,6 +619,9 @@ def _train_epoch(self, train_loader, retain_graph):
             else:
                 metric = "accuracy"
 
+            if self._final_transform:
```

Comment:
sorry, where does `_final_transform` get assigned?

Reply:
in the __init__